xmppserver/src/main/webapp/session-details.jsp

```diff
@@ -266,19 +266,19 @@
         </td>
         <td>
             <%
-                int status = currentSess.getStatus();
-                if (status == Session.STATUS_CLOSED) {
+                Session.Status status = currentSess.getStatus();
+                if (status == Session.Status.CLOSED) {
```

Comment:
Do you want to use the various `if(currentSess.isClosed/Authenticated/etc`) methods here?

Reply:
Maybe, but the reasoning in https://github.com/igniterealtime/Openfire/pull/2165#discussion_r1071926857 applies here too.